### PR TITLE
update Bonita and migration tool version, apply product renaming (rem…

### PR DESCRIPTION
**Pull request by JeremJR**
…ove BPM for 7.6)
related to https://github.com/docker-library/official-images/pull/3800

### CODE REVIEW

**Comment by JeremJR** [comment]
The Travis CI build failed on check-markdownfmt.sh but locally the only thing raised is not related to bonita : 
```
.travis/check-markdownfmt.sh
Need markdownfmt:
./telegraf/content.md

diff ./telegraf/content.md markdownfmt/./telegraf/content.md
--- /tmp/markdownfmt494834690	2017-12-13 16:44:48.840954664 +0000
+++ /tmp/markdownfmt207586681	2017-12-13 16:44:48.840954664 +0000
@@ -171,3 +171,4 @@
 -	[Input Plugins](https://docs.influxdata.com/telegraf/latest/inputs/)
 
 -	[Output Plugins](https://docs.influxdata.com/telegraf/latest/outputs/)
+
```

**Comment by tianon** [comment]
Doh, sorry, that was due to a slight oversight in some of the fixes I made to `markdownfmt` last night -- I've rebuilt the image with a fix and re-triggered the Travis build here.

File: bonita/license.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
-Bonita BPM image includes two parts :
+Bonita image includes two parts :
 
--	Bonita BPM Engine under [LGPL v2.1](https://www.gnu.org/licenses/old-licenses/lgpl-2.1.html)
+-	Bonita Engine under [LGPL v2.1](https://www.gnu.org/licenses/old-licenses/lgpl-2.1.html)
+
+-	Bonita Portal under [GPL v2.0](http://www.gnu.org/licenses/old-licenses/gpl-2.0.html)
```
Comment (tianon) [inline bonita/license.md]: Now `markdownfmt` is only complaining about this single extra blank line. :+1:

Comment (JeremJR) [reply bonita/license.md]: Ok thanks, I've reverted this commit.

**Pull request merged**